### PR TITLE
Add --root-ca-key code to Mesos controller-manager fork

### PR DESCRIPTION
**Pull request by sttts**
This code block was missed during previous rebases. It is necessary to get service accounts working in #10049.

### CODE REVIEW

**Comment by yujuhong** [comment]
@sttts, please remove "WIP" from the subject if the PR is ready for review. Thanks!

File: contrib/mesos/pkg/controllermanager/controllermanager.go [hunk 1]
```diff
@@ -150,6 +152,20 @@ func (s *CMServer) Run(_ []string) error {
...
+		if err != nil {
+			return fmt.Errorf("error reading root-ca-file at %s: %v", s.RootCAFile, err)
+		}
+		if _, err := util.CertsFromPEM(rootCA); err != nil {
+			return fmt.Errorf("error parsing root-ca-file at %s: %v", s.RootCAFile, err)
+		}
+	} else {
+		rootCA = kubeconfig.CAData
+	}
+
```
Comment (tsenart) [inline contrib/mesos/pkg/controllermanager/controllermanager.go]: LGTM, but could be terser:
```go
var rootCA []byte
if rootCA = kubeconfig.CAData; s.RootCAFile != "" {
    if rootCA, err = ioutil.ReadFile(s.RootCAFile); err != nil {
        return fmt.Errorf("error reading root-ca-file at %s: %v", s.RootCAFile, err)
    } else if _, err = util.CertsFromPEM(rootCA); err != nil {
        return fmt.Errorf("error parsing root-ca-file at %s: %v", s.RootCAFile, err)
    }
}
```

Comment (sttts) [inline contrib/mesos/pkg/controllermanager/controllermanager.go]: It's the original code, just copied (as the rest of the controller manager). So I didn't want to touch it, for easier refactoring.

**Comment by sttts** [comment]
@yujuhong it's now ready for review

File: contrib/mesos/pkg/controllermanager/controllermanager.go (hunk 1 above)
Comment (tsenart) [inline contrib/mesos/pkg/controllermanager/controllermanager.go]: SGTM

**Comment by davidopp** [comment]
Waiting for an LGTM from a Mesosphere person

**Comment by tsenart** [comment]
Sorry if it wasn't clear above: LGTM

**Comment by davidopp** [comment]
LGTM

**Pull request merged**